jupyterlab/jupyterlab_jupytext/__init__.py

```diff
@@ -8,6 +8,10 @@
     from jupytext.contentsmanager import build_jupytext_contents_manager_class
 except ImportError as err:
     build_jupytext_contents_manager = reraise(err)
+try:
+    from jupytext.async_contentsmanager import build_jupytext_async_contents_manager_class
+except ImportError as err:
+    build_jupytext_async_contents_manager = reraise(err)
```

Comment:
Yes that's the most prudent approach, thank you! The import problem was years ago though, so maybe it's gone, but it's safer that I follow-up on that later on.